prng/TestGenerators.roc

```diff
@@ -0,0 +1,41 @@
+## Generators for testing RNG implementations
+interface TestGenerators
+    exposes [
+        generateInc,
+        generateConstant,
+        generateCycle,
+    ]
+    imports [PRNG]
+
+generateInc : PRNG.Generator U32 U32
+generateInc = \s -> ((s + 1), s)
+
+expect
+    s0 = 0
+    (s1, x) = generateInc s0 
+    (s2, y) = generateInc s1 
+    (_s, z) = generateInc s2 
+    [x, y, z] == [0, 1, 2]
```

Comment:
Instead of `[x, y, z] == [0, 1, 2]`, `(x, y, z) == (0, 1, 2)` would do the same, but with better performace characteristics, since there is no need to initialize a list on the heap. Ofc it doesn't matter here, but it is a good general rule I think

Reply:
Fixed, thanks for the feedback! :+1: